CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 * [#1216](https://github.com/ruby-grape/grape/pull/1142): Fix JSON error response when calling `error!` with non-Strings - [@jrforrest](https://github.com/jrforrest).
 * [#1225](https://github.com/ruby-grape/grape/pull/1225): Fix `given` with nested params not returning correct declared params - [@JanStevens](https://github.com/JanStevens).
+* [#1227](https://github.com/ruby-grape/grape/pull/1227): Store message_key on validation exception - [@stjhimy](https://github.com/sthimy).
```

Comment:
Quote `message_key` and maybe make the class specific, so Grape::Exceptions::Validation? NBD